lib/modules/commentTools.js

```diff
@@ -1078,24 +1078,6 @@ function lod() {
 	});
 } */
 
-export const KEYS = {
```

Comment:
Can we keep this as `NAMED_KEYS = { SPACE: ' ', ENTER: 'Enter', ESCAPE: 'Escape', ... }` for stronger validation?

Reply:
Sure, sounds good to me.